src/components/common/Extensions/UserHub/partials/NotificationsTab/partials/Notification/Expenditure/ExpenditureNotification.tsx

```diff
@@ -62,14 +64,17 @@ const ExpenditureNotification: FC<NotificationProps> = ({
   const action = actionData?.getColonyAction;
 
   const handleNotificationClicked = () => {
-    if (transactionHash) {
-      navigate(
-        `${window.location.pathname}?${TX_SEARCH_PARAM}=${transactionHash}`,
-        {
-          replace: true,
-        },
-      );
+    if (!transactionHash) {
+      return;
     }
+
+    const path = isCurrentColony
```

Comment:
This is actually quite crafty, so if we're on the current colony, we just open up the sidebar, if we're not, we navigate to that colony and open up the sidebar :100: